src/admin-views/aggregator/status.php

```diff
@@ -176,16 +176,17 @@
 			$notes = esc_html__( 'The service has disabled oAuth. Some types of events may not import.', 'the-events-calendar' );
 		}
 		?>
-        <tr>
-            <td class="label">
-                <img src="<?php echo esc_url( tribe_events_resource_url( 'images/aggregator/eventbrite.png' ) ); ?>" /><span><?php esc_html_e( 'Eventbrite', 'the-events-calendar' ); ?></span>
-            </td>
-            <td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
-            <td><?php echo esc_html( $text ); ?></td>
-            <td><?php echo $notes; ?></td>
-        </tr>
+		<tr>
+			<td class="label">
+				<img src="<?php echo esc_url( tribe_events_resource_url( 'images/aggregator/eventbrite.png' ) ); ?>" /><span><?php esc_html_e( 'Eventbrite', 'the-events-calendar' ); ?></span>
+			</td>
+			<td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
+			<td><?php echo esc_html( $text ); ?></td>
+			<td><?php echo $notes; ?></td>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$notes'

<sub>hash: fee42a82cc8f878be78232e6a2337314</sub>